src/styles/index.scss

```diff
@@ -1,3 +1,108 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid black;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  &__img {
+    margin-block: 0 40px;
+    padding-inline: 3px;
+    object-fit: contain;
+  }
+  &__product {
+    margin-block: 0 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+  &__code {
+    margin-block: 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+  &__rev {
+    margin-block: 0 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-block: 0 16px;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+  }
+  &__cost {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+  }
+  &__button {
+    padding: 12px 70px;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    box-sizing: border-box;
+    background-color: #00acdc;
+    color: white;
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+  }
```

Comment:
look, here you can center a text without padding, this is a better solution(using flexbox): 
```css
&__button {
    display: flex;
    justify-content: center; /* <-- center the `BUY` text with flexbox, not padding */
    align-items: center;
    height: 40px; /* <-- set fixed height (no need to use padding) */

    border: 1px solid #00acdc;
    border-radius: 5px;

    background-color: #00acdc;
    color: white;

    text-transform: uppercase;
    text-decoration: none;
    font-size: 14px;
    font-weight: 700;
    line-height: 16px;
  }
```
also I removed `text-align` and `box-sizing`, because they didn't affect anything